src/main/java/com/google/sps/data/Drink.java

```diff
@@ -0,0 +1,84 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.awt.geom.Point2D;
+import java.util.HashMap;
+
+public class Drink {
+
+  private final String name;
+  private final String id;
+  public HashMap<String, Point2D> storeIDToAvgRating;
```

Comment:
using Point2D is hacky here.  What are the reasons for it? Is there something else you can use?

Reply:
I can use something else.